src/holofuel/pages/CreateOfferRequest/CreateOfferRequest.js

```diff
@@ -237,7 +237,8 @@ export default function CreateOfferRequest ({ history: { push } }) {
 
 export function RenderNickname ({ agentId, setCounterpartyNick, setCounterpartyFound, newMessage }) {
   const { loading, error: queryError, data: { holofuelCounterparty = {} } = {} } = useQuery(HolofuelCounterpartyQuery, {
-    variables: { agentId }
+    variables: { agentId },
+    fetchPolicy: 'network-only'
```

Comment:
Should we be using `no-cache` instead?

Reply:
Also should we just remove this Query? And we can avoid calling the new type `TransactionHolofuelUser` and instead just use `HolofuelUser`. Because we will not need `HolofuelCounterpartyQuery`

Or should we wait for a new PR?

Reply:
That will be the necessary change for the ticket to remove the counterparty check, which I was planning on doing as a separate PR to keep this PR specific to the named ticket number.  Also, the functional updates to the createRequestOffer page will also be paired with UX updates.. so it will be cleaner to keep those changes apart from the focus of this PR.  

That said, the question of whether to link the functional and UX updates to the createRequestOffer page is def a product Q.  Let's follow-up with Rob on that one, and depending on his answer, we might be able to just make a new short PR which is merged into this one.

Thoughts?

Reply:
@JettTech, the only reason I bring this up now is because I think it would be easier to make those changes in the PR.

But I differ to you

Reply:
Checking with Rob.